nearpy/utils/utils.py

```diff
@@ -89,3 +89,9 @@ def want_string(arg, encoding='utf-8'):
     else:
         rv = arg
     return rv
+
+
+def get_keys(engine):
```

Comment:
This function in its current state makes sense only in tests and only for engine with memory storage and unibucket hashes. I initially suggested moving it into method of the test suite (methods with names that are not started with "test_" are not treated as tests and can be used as helpers). Ideally you can make your own suite with custom setup and helpers, but I'm fine with leaving all in the `engine_tests`.